src/utils/VerificationValidationParser.cpp

```diff
@@ -153,9 +156,62 @@ void Parser::searchFinalDefense(Result* r) {
     }
 }
 
-Result* Parser::lc(const QString* terminalOutput) {
-    return nullptr; // TODO: implement
+Result* Parser::lc(const QString* cmd, const QString* terminalOutput) {
+	Result final;
```

Comment:
so in C++, whenever you declare something inside a function, it only lasts for the scope of the function
- after the function ends, whatever was allocated on the function's stack is deallocated

so since `Result final` is declared in function stack, whenever you return the pointer, it'll be pointing to deallocated memory (garbage values)

so solution is to allocate on the heap (e.g.: Result* r = new Result;)
- can refer to search()
this way, whenever the result is returned from the function, it will point to allocated memory on the heap